Tools/scripts/board_list.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+'''
+list of boards for build_binaries.py and custom build server
+'''
+
+AUTOBUILD_BOARDS = ["fmuv2",
```

Comment:
I think it would be better to order these alphabetically

Reply:
easier for the script using them to sort them